Cargo.toml

```diff
@@ -331,13 +331,15 @@ cc = { version = "1.0.37", default-features = false }
 
 [features]
 # These features are documented in the top-level module's documentation.
-default = ["alloc", "dev_urandom_fallback"]
+default = ["alloc", "dev_urandom_fallback", "nasm"]
 alloc = []
 dev_urandom_fallback = ["lazy_static"]
 internal_benches = []
+nasm = []
 slow_tests = []
 std = ["alloc"]
 test_logging = []
+wasm = ["web-sys"]
```

Comment:
Please rebase this MR on master directly, instead of basing it on the other PR you submitted. Then we can deal with them independently.